### PR TITLE
Update the themes documentation 

### PR DESCRIPTION
**Pull request by origami-z**

### CODE REVIEW

File: site/docs/themes/index.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ Salt theme is a collection of defined visual styles that shape the overall look-
 
 A theme in Salt enables delivery teams to streamline the process of creating a consistent design language across their business domain, platforms or products.
 
-### Design tokens
+### Design Tokens
```
Comment (origami-z) [inline site/docs/themes/index.mdx:15]: Sentence case for heading is part of content guide

Comment (joshwooding) [reply site/docs/themes/index.mdx:15]: "Design Tokens" is title case, sentence case is where only the first letter of the sentence is capitalised.

**Comment by amritadesmet** [comment]
**Changes made:** 

- Consistent use of title case throughout page 
- Changed all instances of the word "legacy" to "current theme"

**Review by Fercas123 — APPROVED** [review]

File: site/docs/themes/index.mdx [hunk 2]
```diff
@@ -167,13 +167,13 @@ Optional border support is already being rolled out across Salt's form control c
   alt="Input border styling options"
 />
 
-## JPM Brand endorsed
+## JPM brand endorsed
```
Comment (joshwooding) [inline site/docs/themes/index.mdx:170]:
```suggestion
## JPM brand endorsement
```

File: site/docs/themes/index.mdx [hunk 3]
```diff
@@ -276,7 +276,7 @@ To access the libraries:
 
 4. Add the three available libraries (referenced above) to your file.
 
-#### Using the Style Library
+#### Using the style Library
```
Comment (joshwooding) [inline site/docs/themes/index.mdx:279]:
```suggestion
#### Using the style library
```

File: site/docs/themes/index.mdx [hunk 4]
```diff
@@ -276,7 +276,7 @@ To access the libraries:
...
 
 The Style Library includes all the Figma variables needed to design for Salt. It includes the semantic palette, category colors, size and spacing values, corner radiuses, typographic styles and more. Using Style Library variables to style your components automatically adds 'styling options' to your components.
```
Comment (joshwooding) [inline site/docs/themes/index.mdx:281]:
```suggestion
The style library includes all the Figma variables needed to design for Salt. It includes the semantic palette, category colors, size and spacing values, corner radiuses, typographic styles and more. Using Style Library variables to style your components automatically adds 'styling options' to your components.
```

File: site/docs/themes/index.mdx [hunk 5]
```diff
@@ -287,13 +287,13 @@ The variables are set up to match our tokens in code. When inspecting a design i
   alt="Figma next library next dev mode"
 />
 
-#### Using the Colors Library
+#### Using the colors library
 
 The Colors Library provides access to all foundational color ramps (e.g., `--blue-100` to `--blue-900`). These colors are linked to the semantic styles in the Salt Next Styles library.
```
Comment (joshwooding) [inline site/docs/themes/index.mdx:292]:
```suggestion
The colors library provides access to all foundational color ramps (e.g., `--blue-100` to `--blue-900`). These colors are linked to the semantic styles in the Salt Next Styles library.
```

File: site/docs/themes/index.mdx [hunk 6]
```diff
@@ -287,13 +287,13 @@ The variables are set up to match our tokens in code. When inspecting a design i
...
 
 You don't need to add these colors to your project unless you want to use them directly. For consistency across components and to benefit from 'styling options', Salt recommends only applying variables from the [Salt Next Style library](#using-the-style-library) to your project.
 
-#### Using the Component & Pattern Library
+#### Using the component & pattern library
```
Comment (joshwooding) [inline site/docs/themes/index.mdx:296]:
```suggestion
#### Using the component and pattern library
```

File: site/docs/themes/index.mdx [hunk 7]
```diff
@@ -287,13 +287,13 @@ The variables are set up to match our tokens in code. When inspecting a design i
...
 
 The new variable-driven Component & Pattern library functions similarly to the current Salt component library but with key differences:
```
Comment (joshwooding) [inline site/docs/themes/index.mdx:298]:
```suggestion
The new variable-driven component and pattern library functions similarly to the current Salt component library but with key differences:
```

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**